bin/embedding-sdk/generate-sdk-package-files.js

```diff
@@ -85,4 +89,5 @@ if (!fs.existsSync(SDK_DIST_DIR)) {
 }
 
 generateSdkPackage();
-generateLicenseFile();
+copyFileToOutput("LICENSE.txt");
+copyFileToOutput("frontend/src/embedding-sdk/README.md", "README.md");
```

Comment:
We need to show the README in the NPM registry page. Otherwise it shows "no README".